subworkflows/local/repeat_analysis.nf

```diff
@@ -34,33 +34,14 @@ workflow REPEAT_ANALYSIS {
 
     BCFTOOLS_SORT_TRGT.out.vcf
         .join(BCFTOOLS_INDEX_TRGT.out.csi)
-        .set{ ch_bcftools_query_in }
-
-    ch_bcftools_query_in
-        .map{[['id':'multisample'],it[1]]}
-        .groupTuple()
-        .set{ vcfs }
-
-    ch_bcftools_query_in
-        .map{[['id':'multisample'],it[2]]}
+        .toList()
+        .filter { it.size() > 1 }
```

Comment:
nice, haven't used the filter operator much. Usually been handling this type of operations by making two branches and running merge on the one with multiple samples and after that mixing the two channels. 
https://github.com/nf-core/raredisease/blob/f93c357a4b164031197c22356604a2a08026c31d/subworkflows/local/alignment/align_bwa_bwamem2.nf#L51

https://github.com/nf-core/raredisease/blob/f93c357a4b164031197c22356604a2a08026c31d/subworkflows/local/alignment/align_bwa_bwamem2.nf#L59

Reply:
Nice! I'll keep that in mind, I think your solution looks cleaner in most cases.